docker/ethdebug.go

```diff
@@ -0,0 +1,24 @@
+package scale
```

Comment:
Shouldn't it be `package docker` here?

Reply:
i am not sure that these tests should stay in this directory